qemu/tests/fio_windows.py

```diff
@@ -34,6 +36,12 @@ def run(test, params, env):
     vm = env.get_vm(params["main_vm"])
     vm.verify_alive()
     session = vm.wait_for_login(timeout=timeout)
+    session_1 = vm.wait_for_login(timeout=timeout)
```

Comment:
it this session for fio test on data disk? please give a meaningful  name here

Reply:
thanks, due to these 2 sessions are uesed to run data disk fio and sys disk fio test, so changed the name to session_data and session_sys.